packages/client/src/pages/Forum/ForumTopic/index.tsx

```diff
@@ -31,18 +26,15 @@ export const ForumTopic = () => {
 	}
 
 	function onSubmitHandler() {
-		if (newMessage.value) {
-			setMessages(prev => [...prev, {
-				id: prev[prev.length - 1].id + 1,
-				isOwner: true,
-				text: newMessage.value,
-			}]);
-
-			newMessage.nulling();
+		if (newMessage.value.trim()) {
+			dispatch(forumActions.addMessage(
+				newMessage.value.trim(),
+			));
```

Comment:
повторяющееся действие прямо просится вынести его:
```suggestion
    const messageContent = newMessage.value.trim();
		if (messageContent) {
			dispatch(forumActions.addMessage(
				messageContent,
			));
```